test/unit/browser-test.js

```diff
@@ -0,0 +1,54 @@
+define([
+    'utils/browser'
+], function (browser) {
+    /* jshint qunit: true */
+
+    module('browser');
+
+    var testerGenerator = function (assert, method) {
+        return function (left, right, message) {
+            assert.strictEqual(method.apply(this, left), right, message);
+        };
+    };
+
+    test('browser.isInt', function(assert) {
```

Comment:
Surprise! Perhaps we should make isInt a private method within this utils file, since it doesn't seem related to browser checking.